src/CifFile_module.py

```diff
@@ -3077,16 +3077,15 @@ def validate_item_esd(self,item_name,item_value):
         return {"result":True}
 
     def validate_item_esd_ddlm(self,item_name,item_value):
-        if self[item_name].get('self.primitive_type') not in \
+        if self[item_name].get(self.primitive_type) not in \
```

Comment:
It appears that this method has never been executed given there was an error in the first line.